### ISSUE TITLE
nan from autograd exp/log function in spd manifold (pytorch backend)

### ISSUE THREAD

**Issue by fangyihaoo** [geomstats/geomstats#982]
Hi, thanks for implementing those manifold structures in python!

I'm working with SPD data and I tried autograd to calculate the gradient of exp/log function with respect to the base_point in spd manifold (pytorch backend), however, I got nan value in the gradient of the base point (b.grad) after running the code below :

import geomstats.geometry.spd_matrices as spd
le_metric = spd.SPDMetricLogEuclidean(2)
a = torch.tensor([[2.,1.],[1.,2.]],requires_grad=True) 
b = torch.eye(2,requires_grad=True)
c = le_metric.exp(a,base_point = b)
d = c.sum()
d.backward()
b.grad

Could you guys help me to fix this problem?

**Comment by nguigs** [comment]
Hi @fangyihaoo and thanks for your feedback! This is related to #966, we are currently working on it.
